.gitignore

```diff
@@ -0,0 +1,7 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+/.vs/BitsOfReviewBot/v15/.suo
```

Comment:
If you want to remove .vs/BitsOfReviewBot/v15.suo, you'll need to remove the cached. The files are still being commited, so it needs "git rm --cached .vs/BitsOfReviewBot/v15/.suo"

Reply:
Because of all the vs files, it might just be useful to do a git ignore of .vs/*